engine-src/Game/LambdaHack/Client/UI/HandleHumanGlobalM.hs

```diff
@@ -927,6 +927,45 @@ applyItem (fromCStore, (iid, (itemFull, kit))) = do
         modifySession $ \sess -> sess {sitemSel = Nothing}
         failWith "never mind"
 
+-- * CloseDir 
+
+-- | Close nearby open tile. Ask for direction, if there is more than one.
+closeDirHuman :: MonadClientUI m
+              => m (FailOrCmd RequestTimed)
+closeDirHuman = do
+  COps{cotile} <- getsState scops
+  leader <- getLeaderUI
+  b <- getsState $ getActorBody leader
+  lvl <- getLevel $ blid b
+  let vPts = vicinityUnsafe $ bpos b 
+      openPts = filter (Tile.isClosable cotile . at lvl) vPts
```

Comment:
Well done. The command is about closing and we check `isClosable`. For other terrain modifications there's the 'M' command.